doc/neps/nep-0016-zero-rank-arrarys.rst

```diff
@@ -0,0 +1,238 @@
+=========================
+NEP 16 — Zero Rank Arrays
+=========================
+
+:Author: Alexander Belopolsky (sasha), transcribed Matt Picus <matti.picus@gmail.com>
+:Status: Draft
+:Type: Informational
+:Created: 2018-10-14
```

Comment:
I wonder if we want to backdate this

Reply:
Yeah, let's put the original date here -- maybe noting the date it was transcribed, too?

Reply:
ok